account_central_billing/views/res_partner.xml

```diff
@@ -7,7 +7,7 @@
         <field name="inherit_id" ref="account.view_partner_property_form"/>
         <field name="arch" type="xml">
             <xpath expr="//field[@name='website']" position="after">
-                <field name="store_ref" attrs="{'invisible': [('is_company', '=', False)]}"/>
+                <field name="store_ids" attrs="{'invisible': [('is_company', '=', False)]}"/>
```

Comment:
These are most definitely different fields. Something is wrong here. Store Ref is a simple char field basically a code assigned to a store. I have no idea what store_ids is, but it isn't the same.